django_elastic_migrations/models.py

```diff
@@ -965,7 +965,14 @@ def __init__(self, *args, **kwargs):
         super(UpdateIndexAction, self).__init__(*args, **kwargs)
         if self.task_kwargs == '{}' and self.self_kwargs:
             # retain a history of how this command was called
-            self.task_kwargs = json.dumps(self.self_kwargs, sort_keys=True)
+            try:
+                self.task_kwargs = json.dumps(self.self_kwargs, sort_keys=True)
+            except TypeError as e:
+                if 'Object of type datetime is not JSON serializable' in str(e):
+                    for key, val in self.self_kwargs.items():
+                        if isinstance(val, datetime.date):
+                            self.self_kwargs[key] = str(val)
```

Comment:
Judging from `kwarg_defaults` above, it seems like these two lines could be slightly modified to specifically reference the `'start_date'` key and moved above, then there wouldn't need to be a `try` block.

Reply:
good point, it's better to be explicit. I'll update.